brave/src/main/java/com/linecorp/armeria/client/brave/BraveClient.java

```diff
@@ -57,109 +50,65 @@
 
     private static final Logger logger = LoggerFactory.getLogger(BraveClient.class);
 
-    // TODO(minwoox) Add the variant which takes HttpTracing.
-
-    /**
-     * Creates a new tracing {@link Client} decorator using the specified {@link Tracing} instance.
-     */
-    public static Function<Client<HttpRequest, HttpResponse>, BraveClient> newDecorator(Tracing tracing) {
-        return newDecorator(tracing, null);
-    }
-
     /**
-     * Creates a new tracing {@link Client} decorator using the specified {@link Tracing} instance
-     * and remote service name.
+     * Creates a new tracing {@link Client} decorator using the specified {@link HttpTracing} instance.
      */
     public static Function<Client<HttpRequest, HttpResponse>, BraveClient> newDecorator(
-            Tracing tracing, @Nullable String remoteServiceName) {
+            HttpTracing httpTracing) {
```

Comment:
I think we can create a default `HttpTracing` from `Tracing` and `String` - even if we don't want that to be part of the public API going forward it would be good to do so with deprecation at least.